r/tests/testthat/test-compute.R

```diff
@@ -103,12 +105,14 @@ test_that("register_scalar_function() adds a compute function to the registry",
     Scalar$create(32L, float64())
   )
 
-  expect_identical(
-    record_batch(a = 1L) %>%
-      dplyr::mutate(b = times_32(a)) %>%
-      dplyr::collect(),
-    tibble::tibble(a = 1L, b = 32.0)
-  )
+  withr::with_envvar(list(R_ARROW_COLLECT_WITH_UDF = "true"), {
```

Comment:
Can revert this

Reply:
I was thinking you can revert this because you've already called `register_scalar_function()` so the env var will be set.